auto3dseg/notebooks/msd_datalist_generator.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "import random"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "outputs": [],
+   "source": [
+    "# Construct skeleton JSON to populate with your own data\n",
+    "datalist_json = {\n",
+    "    \"testing\": [],\n",
+    "    \"training\": []\n",
+    "}"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "outputs": [],
+   "source": [
+    "# Populate testing with files in your directory\n",

```

Comment:
Is it possible to add a cell that downloads the data to `MONAI_DATA_DIRECOTORY` or a temp folder?

```python
directory = os.environ.get("MONAI_DATA_DIRECTORY")
root_dir = tempfile.mkdtemp() if directory is None else directory
print(root_dir)
```

At the end of the notebook, we should also remove the folder if it is temporary
```python
if directory is None:
    shutil.rmtree(root_dir)
```